extensions/azurecore/src/azureResource/providers/database/databaseService.ts

```diff
@@ -48,8 +58,8 @@ export class AzureResourceDatabaseService implements IAzureResourceService<azure
 						name: db.name,
 						id: db.id,
 						serverName: server.name,
-						serverFullName: server.properties.fullyQualifiedDomainName,
-						loginName: server.properties.administratorLogin,
+						serverFullName: server.constructor.name === 'DbSynapseGraphData' ? (server as DbSynapseGraphData).properties.connectivityEndpoints.sql : (server as DbServerGraphData).properties.fullyQualifiedDomainName,
```

Comment:
You should use `instanceof` instead of the constructor value if you want to check if it's a specific class type. But these are interfaces - not classes. So I'm confused how this is working in the first place...

But you should just be able to do something like `(server as DbSynapseGraphData).properties.connectivityEndpoints?.sql ?? (server as DbServerGraphData).properties.fullQualifiedDomainName`

And then a similar thing for login name.